src/Payments.sol

```diff
@@ -231,10 +248,17 @@ contract Payments is
         account.funds += amount;
     }
 
+    /// @notice Withdraws tokens from the caller's account to the caller's account.
+    /// @param token The ERC20 token address to withdraw.
+    /// @param amount The amount of tokens to withdraw.
     function withdraw(address token, uint256 amount) external nonReentrant {
         return withdrawToInternal(token, msg.sender, amount);
     }
 
+    /// @notice Withdraws tokens (`token`) from the caller's account to `to`.
+    /// @param token The ERC20 token address to withdraw.
+    /// @param to The address to receive the withdrawn tokens.
+    /// @param amount The amount of tokens to withdraw.
```

Comment:
can only withdraw unlocked funds after account lockup is settled and only if account lockup is fully settled*.